doc/generate_logos.py

```diff
@@ -215,14 +234,19 @@ def convert_to_ico(fn_source, output_dir, sizes):
         pngs = []
         for size in sizes:
             fn_png= "%s-%dpx.png" % (name, size)
-            fn_png = os.path.join(output_dir, fn_png)
+            if system()[0:3].lower() != "win":
+                fn_png = os.path.join(output_dir, fn_png)
             pngs.append(fn_png)
 
         # convert them to *.ico
         fn_out = "%s-favicon.ico" % name
-        fn_out = os.path.join(output_dir, fn_out)
+        if system()[0:3].lower() != "win":
+            fn_out = os.path.join(output_dir, fn_out)
 
-        cmd = "convert {} {}".format(" ".join(pngs), fn_out)
+        if system()[0:3].lower() == "win":
+            cmd = "magick {} {}".format(" ".join(pngs), fn_out)
+        else:
+            cmd = "convert {} {}".format(" ".join(pngs), fn_out)
```

Comment:
This if-else is not needed as you previously defined cmd according to os.

Reply:
Sorry for the delay, will get it done ASAP